tests/dict_expr_01.py

```diff
@@ -0,0 +1,4 @@
+def test_dict():
+    mydictionarty3: dict[int, float] = {}
```

Comment:
This is currently not supported in LPython, as we do not allow arbitrary precision integers.